### PR TITLE
doc: updated instruction manual to set-up a CI server

### PR DESCRIPTION
**Pull request by leemgs**
This commit is to update the existing instruction manual to give users
more easy setting guide when developers try to set-up a sever environment
to run CI server newly.

**Changes proposed in this PR:**
1. Updated the existing server setting guide

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

---

### CODE REVIEW

File: ci/doc/how-to-setup-taos-ci-server.md [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+# chown -R www-data:www-data /var/www/
+# cp /root/.bashrc /var/www/
+# chown -R www-data:www-data  /var/www/.bashrc
+# exit
+$
+```
+
 ## Prerequisites
 We assume that you already installed Ubuntu 16.04 x86_64 distribution in your own computer.
+* For a physical machine, http://mirror.kakao.com/ubuntu-releases/xenial/
```
Comment (leemgs) [inline ci/doc/how-to-setup-taos-ci-server.md]: There are three methods to install an Ubuntu Linux distribution.

File: ci/doc/how-to-setup-taos-ci-server.md [hunk 2]
```diff
@@ -281,3 +268,20 @@ sudo chown -R www-data:www-data /opt/scancode-toolkit/
 mkdir  /var/www/html/{your_prj_name}/scancode/
 /opt/scancode-toolkit/scancode  --license /var/www/html/{your_prj_name}/{src_folder}  --html-app /var/www/html/{your_prj_name}/scancode/index.html
 ```
+
```
Comment (leemgs) [inline ci/doc/how-to-setup-taos-ci-server.md]: It's is optional guide.

File: ci/doc/how-to-setup-taos-ci-server.md [hunk 3]
```diff
@@ -1,7 +1,25 @@
...
+# chown -R www-data:www-data  /var/www/.bashrc
+# exit
+$
+```
+
 ## Prerequisites
 We assume that you already installed Ubuntu 16.04 x86_64 distribution in your own computer.
+* For a physical machine, http://mirror.kakao.com/ubuntu-releases/xenial/
+* For a virtual machine, https://www.osboxes.org/ubuntu/
+* For a docke image, https://hub.docker.com/_/ubuntu/
```
Comment (ohsewon) [inline ci/doc/how-to-setup-taos-ci-server.md]: typo docke -> docker

Comment (leemgs) [reply ci/doc/how-to-setup-taos-ci-server.md]: Fixed. resubmitted.

**Pull request merged**